services/skus/datastore.go

```diff
@@ -796,8 +796,16 @@ func (pg *Postgres) DeleteSingleUseOrderCredsByOrderTx(ctx context.Context, tx *
 
 // DeleteTimeLimitedV2OrderCredsByOrderTx performs a hard delete for all time limited v2 order
 // credentials for a given OrderID.
-func (pg *Postgres) DeleteTimeLimitedV2OrderCredsByOrderTx(ctx context.Context, tx *sqlx.Tx, orderID uuid.UUID) error {
-	_, err := tx.ExecContext(ctx, `delete from time_limited_v2_order_creds where order_id = $1`, orderID)
+func (pg *Postgres) DeleteTimeLimitedV2OrderCredsByOrderTx(ctx context.Context, tx *sqlx.Tx, orderID uuid.UUID, itemIDs ...*uuid.UUID) error {
+	// legacy, if the request id matches the item id on the time limited v2 creds, we actually want to delete the record,
+	// otherwise we will keep these credentials here for multiple device refresh capabilities
+	query, args, err := sqlx.In("delete from time_limited_v2_order_creds where order_id = ? and item_id in (?)", orderID, itemIDs)
```

Comment:
issue: question mark is a placeholder in MySQL. We use Postgres where numerated placeholders are used.

Reply:
Checked the docs. Found why this should use `?`. Resolved.